yt_dlp/extractor/nebula.py

```diff
@@ -195,44 +128,156 @@ def _real_initialize(self):
         if nebula_cookie:
             self.to_screen('Authenticating to Nebula with token from cookie jar')
             nebula_cookie_value = compat_urllib_parse_unquote(nebula_cookie.value)
-            self._nebula_token = self._parse_json(nebula_cookie_value, None).get('apiToken')
+            self._nebula_api_token = self._parse_json(nebula_cookie_value, None).get('apiToken')
 
         # try to authenticate using credentials if no valid token has been found
-        if not self._nebula_token:
-            self._nebula_token = self._retrieve_nebula_auth()
+        if not self._nebula_api_token:
+            self._nebula_api_token = self._retrieve_nebula_auth()
+
+        # get a Bearer token for the Nebula API, that we need to fetch meta data
+        self._nebula_bearer_token = self._fetch_nebula_bearer_token()
+
+        # get a Zype access token which is required to access video streams (for us: to generate video URLs)
+        self._zype_access_token = self._fetch_zype_access_token()
```

Comment:
I think it makes sense to move these comments to the docstrings of these functions instead